theme/tutorial-built-in-themes.asciidoc

```diff
@@ -85,22 +94,10 @@ https://github.com/vaadin/vaadin-button/blob/master/theme/lumo/vaadin-button.htm
 The themed files should be stored to
 `${frontend.working.directory}/bower_components/{component}/theme/myTheme` which would by default for vaadin-button be `src/main/webapp/frontend/bower_components/vaadin-button/theme/myTheme/vaadin-button.html`
 
+If you need to import some theme-specific files, use `@HtmlImport` as shown in the example above:
+ `@HtmlImport("frontend://bower_components/vaadin-lumo-styles/color.html")`
 
-If you need in your theme to import some files that are found using for instance the uri scheme `frontend://`
-then AbstractTheme has a method override `getHeadInlineContents(VaadinUriResolver resolver)`
-which makes it possible to resolve the uri scheme.
-
-.Import link for a "frontend://" file
-[source,java]
-----
-@Override
-public List<String> getHeadInlineContents(VaadinUriResolver resolver) {
-    String colorUrl = resolver.resolveVaadinUri("frontend://bower_components/vaadin-lumo-styles/color.html");
-    List<String> contents = new ArrayList<>();
-    contents.add("<link rel=\"import\" href=\"" + colorUrl + "\">");
-    return contents;
-}
-----
+Each of those html imports will be inlined to the BootstrapPage head, same as html from `getBodyInlineContents()`.
```

Comment:
Also here I guess we should note that the file contents are not inlined? (at least they weren't)
So perhaps just say that `... will be added to the BootstrapPage head.`